### ISSUE TITLE
Enhancement: Compatibility with Ex Compressum

### ISSUE THREAD

**Issue by MadRatSRP** [Sediments/Compressions#12]
If there a chance to see that thing: if ex compressum is found, then compressions adds its blocks in config, but in minecraft they are initialized as a Compressions blocks, with attributes of both mods. 
Here what i mean. U open minecraft with blocks, u have compressed blocks x1. u can interract with them using hammer from ex nihilo. U close minecraft and can change their value of compression. thats the first thing.
The second thing is :if first thing is working, then the second one probably will work. Using of compressions of higher level in heavy sieve(ex compressum item) with same procent of efficiency as a compressions lvl 1 used.

**Comment by Sediments** [comment]
Yeah, compatibility with Ex Compressum is on my list. (When this mod was private, a couple of months ago, I did exactly that, used the compressed blocks with Ex Compressum)

I'm currently rewriting the mod so after that I'll implement this (So after a couple of days).

**Comment by MadRatSRP** [comment]
Nice! 
p.s. if u like anime and smart fights, watch Jojo Bizzare Adventure, really nice thing.
